docs/pages/database-access/guides/sql-server-ad.mdx

```diff
@@ -0,0 +1,324 @@
+---
+title: Database Access with Microsoft SQL Server with Active Directory authentication (Preview)
+description: How to configure Teleport Database Access with Microsoft SQL Server with Active Directory authentication.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="9.0"
+>
+  Database access for Microsoft SQL Server with Active Directory authentication
+  is available starting from Teleport `9.0`.
+</Details>
+
+<Admonition type="warning" title="Preview">
+  Database access for Microsoft SQL Server is currently in a Preview mode and
+  does not include audit logging of database query activity.
+</Admonition>
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Set up access to SQL Server using Active Directory authentication.
+- Connect to SQL Server through Teleport.
+
+This guide will focus on Amazon RDS for SQL Server using AWS-managed Active
+Directory authentication.
+
+## Prerequisites
+
+- Teleport version >= `9.0`.
+- A SQL Server database with Active Directory authentication enabled.
+- A Windows machine joined to the same Active Directory domain as the database.
+- A Linux node joined to the same Active Directory domain as the database. This
+  guide will walk you through the joining steps if you don't have one.
+
+## Step 1/7. Set up Teleport Auth and Proxy
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+## Step 2/7. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+## Step 3/7. Join the Linux node to Active Directory
+
+<Admonition type="note">
+  You can skip this step if you already have a Linux node joined to the same
+  Active Directory domain as your SQL Server.
+</Admonition>
+
+The Linux node where the Database Service will run must be joined to the same
+Active Directory domain as the SQL Server database.
+
+Note that in order to be able to join, the Linux node must be able to resolve
+your Active Directory fully-qualified domain name. For example, for AWS-managed
+AD, use nameservers provided under "Networking details" on the directory's
+overview page.
+
+Install necessary packages:
+
+<Tabs>
+  <TabItem label="Ubuntu">
+    ```bash
+    $ sudo apt-get update
+    $ sudo apt-get -y install sssd realmd krb5-user samba-common packagekit adcli
+    ```
+  </TabItem>
+  <TabItem label="RHEL / CentOS 7">
+    ```bash
+    $ sudo yum -y update
+    $ sudo yum -y install sssd realmd krb5-workstation samba-common-tools
+    ```
+  </TabItem>
+</Tabs>
+
+Edit `/etc/krb5.conf` to disable reverse DNS resolution and set the default
```

Comment:
Can we explicitly underline that  [realms] section should be also updated to point to default real ? 
```
[libdefaults]
 rdns = false
 default_realm = CROP.EXAMPLE.COM

[realms]
 CORP.EXAMPLE.COM = {
  kdc = corp.example.com
  admin_server = corp.example.com
 }
 ```